### PR TITLE
Better Approximate FFTView

### PR DESCRIPTION
**Pull request by emurray2**
These changes make sure the `bufferSize` in `FFTTap` match closely to the `numberOfBars` in `FFTView`.

### CODE REVIEW

File: Sources/AudioKitUI/Visualizations/FFTView.swift [hunk 1]
```diff
@@ -34,26 +34,24 @@ class FFTModel: ObservableObject {
...
-                                          stop2: 1.0)
-                if mappedAmplitude > 1.0 {
-                    mappedAmplitude = 1.0
-                }
-                if mappedAmplitude < 0.0 {
-                    mappedAmplitude = 0.0
-                }
-
-                tempAmplitudes.append(mappedAmplitude)
+        for i in stride(from: 0, to: FFT_SIZE - 1, by: 1) {
```
[the remark this answers is not in the document]
Comment (emurray2) [reply Sources/AudioKitUI/Visualizations/FFTView.swift:37]: Already was named `i`

File: Sources/AudioKitUI/Visualizations/FFTView.swift [hunk 2]
```diff
@@ -4,18 +4,18 @@ import AudioKit
 import SwiftUI
 
 class FFTModel: ObservableObject {
-    @Published var amplitudes: [Double?] = Array(repeating: nil, count: 50)
+    @Published var amplitudes: [Double?] = Array(repeating: nil, count: 64)
     var nodeTap: FFTTap!
-    private var FFT_SIZE = 2048
+    private var FFT_SIZE = 64
```
[the remark this answers is not in the document]
Comment (emurray2) [reply Sources/AudioKitUI/Visualizations/FFTView.swift:9]: Already was named `FFT_SIZE`

**Comment by emurray2** [comment]
I've also noticed one problem with reducing the size of the FFT buffer seems to cause more frequency leakage in the low end?

I'm open to what others think as well. It's been a while since I learned about FFT windows, so I'll need some time to go back and refresh my knowledge.

**Comment by aburgel** [comment]
@emurray2 i tried a different approach here: https://github.com/AudioKit/AudioKitUI/pull/25

I kept the same sized buffer but aggregated the frequencies together so they fit in the desired number of bars.

**Pull request closed without merging**